chromium_src/ios/chrome/browser/shared/model/prefs/browser_prefs.mm

```diff
@@ -34,6 +36,8 @@ void BraveRegisterBrowserStatePrefs(
 #if BUILDFLAG(ENABLE_IPFS)
   ipfs::IpfsService::RegisterProfilePrefs(registry);
 #endif
+  ai_chat::prefs::RegisterProfilePrefs(registry);
+  ai_chat::prefs::RegisterProfilePrefsForMigration(registry);
```

Comment:
I don't see this called in desktop, is it needed?

Reply:
I removed this